Dockerfile.fat

```diff
@@ -46,39 +46,41 @@ RUN echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/main" | tee -a /et
     echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories && \
     apk upgrade --no-cache -a && \
     apk add --no-cache \
-        ca-certificates \
-        tzdata \
-        tini \
-        bash \
-        curl \
-        shadow \
-        su-exec \
-        openssl \
-        openssl-dev \
-        openjdk21-jre \
-# Doc conversion
-		gcompat \
-    	libc6-compat \
-        libreoffice \
-# pdftohtml
-        poppler-utils \
-# OCR MY PDF (unpaper for descew and other advanced featues)
-        qpdf \
-        tesseract-ocr-data-eng \
-        font-terminus font-dejavu font-noto font-noto-cjk font-awesome font-noto-extra \
-# CV
-		py3-opencv \
-# python3/pip
-        python3 \
-    py3-pip && \
-# uno unoconv and HTML
-    pip install --break-system-packages --no-cache-dir --upgrade unoconv WeasyPrint pdf2image pillow && \
+    ca-certificates \
+    tzdata \
+    tini \
+    bash \
+    curl \
+    shadow \
+    su-exec \
+    openssl \
+    openssl-dev \
+    openjdk21-jre \
+    # Doc conversion
+    gcompat \
+    libc6-compat \
+    libreoffice \
+    # pdftohtml
+    poppler-utils \
+    # OCR MY PDF (unpaper for descew and other advanced featues)
+    qpdf \
+    tesseract-ocr-data-eng \
+    font-terminus font-dejavu font-noto font-noto-cjk font-awesome font-noto-extra \
+    # CV
+    py3-opencv \
+    # python3/pip
+    python3 \
+    -X https://dl-cdn.alpinelinux.org/alpine/edge/testing py3-unoconv \
```

Comment:
Why use full URL here, should be able to do @testing etc for labelling since we already have those added